### PR TITLE
feat: Added observability API docs

### PR DESCRIPTION
**Pull request by JigarJoshi**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -12,7 +12,7 @@
     "serve": "docusaurus serve",
     "write-translations": "docusaurus write-translations",
     "write-heading-ids": "docusaurus write-heading-ids",
-    "generate": "protoc --doc_out=./fixtures --doc_opt=json,proto_workspace.json --proto_path=external/tigris-api/ server/v1/api.proto; npx docusaurus generate-proto-docs; npx prettier -w .; sed -i 's/Files/RPC/g' generatedSidebarsProtodocs.js"
+    "generate": "protoc --doc_out=./fixtures --doc_opt=json,proto_workspace.json --proto_path=external/tigris-api/ server/v1/api.proto server/v1/observability.proto; npx docusaurus generate-proto-docs; npx prettier -w .; sed -i 's/Files/RPC/g' generatedSidebarsProtodocs.js"
```
Comment (ovaistariq) [inline package.json:15]: Add the following as well:
- admin.proto
- auth.proto
- health.proto

File: protodocs/server/v1/observability.proto.mdx [hunk 2]
```diff
@@ -0,0 +1,324 @@
...
+      { name: "MIN", number: "1", description: "" },
+      { name: "MAX", number: "2", description: "" },
+      { name: "SUM", number: "3", description: "" },
+    ],
+  }}
+/>
+
+---
+
+## Services
```
Comment (ovaistariq) [inline protodocs/server/v1/observability.proto.mdx:285]: Move this Services section to be above the Messages section

File: generatedSidebarsProtodocs.js [hunk 3]
```diff
@@ -2,12 +2,15 @@ module.exports = {
   protodocs: [
     {
       type: "category",
-      label: "RPC",
+      label: "Files",
```
Comment (ovaistariq) [inline generatedSidebarsProtodocs.js:5]: RPC

**Review by ovaistariq — COMMENTED** [review]
The docusaurus.config.js file also needs to be updated. The `redocusaurus` config key.

File: protodocs/server/v1/observability.proto.mdx (hunk 2 above)
Comment (JigarJoshi) [reply protodocs/server/v1/observability.proto.mdx]: moved

File: generatedSidebarsProtodocs.js (hunk 3 above)
Comment (JigarJoshi) [reply generatedSidebarsProtodocs.js]: fixed sed command

File: package.json (hunk 1 above)
Comment (JigarJoshi) [reply package.json]: added

**Comment by adilansari** [comment]
Merged as part of another commit, hence closing.

**Pull request closed without merging**